core/README.md

```diff
@@ -1,12 +1,16 @@
 # STAC API - Core Specification
 
 - [STAC API - Core Specification](#stac-api---core-specification)
+  - [Summary](#summary)
+  - [Overview](#overview)
   - [Link Relations](#link-relations)
   - [Endpoints](#endpoints)
   - [Example Landing Page for STAC API - Core](#example-landing-page-for-stac-api---core)
   - [Extensions](#extensions)
   - [Structuring Catalog Hierarchies](#structuring-catalog-hierarchies)
 
+## Summary
```

Comment:
I added these headers so the Visual Studio Code Markdown plugin would stop removing the summary section because it thought it was part of the TOC. Same for other files.